src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -189,6 +193,27 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
             // default BG color.
             _settings.DefaultBackground(ARGB(0, R, G, B));
         }
+        else if (!_settings.BackgroundImage().empty())
+        {
+            Windows::Foundation::Uri imageUri{ _settings.BackgroundImage() };
+            Media::Imaging::BitmapImage image(imageUri);
+
+			// Note that BitmapImage handles the image load asynchronously,
+			// which is especially important since the image 
+			// may well be both large and somewhere out on the
+			// internet.
+
+            Media::ImageBrush brush{};
+            brush.ImageSource(image);
+            brush.Stretch(static_cast<Media::Stretch>(_settings.BackgroundImageStretchMode()));
+            brush.Opacity(_settings.BackgroundImageOpacity());
+
+            _root.Background(brush);

```

Comment:
If we introduce another layer just to hold the image, behind the control but within the content root, we probably get both acrylic and background image together for “free”!

Reply:
This seems like a good next step, yes.